src/main.module.ts

```diff
@@ -28,6 +35,38 @@ import { LoggingInterceptor } from './helpers/logging-interceptor.helper';
               singleLine: true,
             },
           },
+          // TODO APIM-222: Do we need to do this for MDM also?
+          // TODO APIM-222: Are the base URLs for ACBS and FIS IdP secret?
```

Comment:
@avaitonis Do you know if we should treat the base URLs for ACBS and FIS IdP as secrets as well? We're logging them currently

Reply:
"base URLs for ACBS and FIS IdP" are private, but maybe not secret and can stay in the logs.

FYI @abhi-markan @IgnatG

Reply:
@abhi-markan @IgnatG if you check the GitHub variables for TFS, you can see the values we're talking about (see the `ACBS_AUTHENTICATION_BASE_URL` and `ACBS_BASE_URL` variables)